src/include/libpmemobj/base.h

```diff
@@ -211,6 +209,19 @@ typedef int (*pmemobj_constr)(PMEMobjpool *pop, void *ptr, void *arg);
  */
 void _pobj_debug_notice(const char *func_name, const char *file, int line);
 
+#ifdef _WIN32
+#ifdef UNICODE
+#define pmemobj_errormsg pmemobj_errormsgW
```

Comment:
All API functions should look and behave exactly the same on both linux and windows. In this case in are changing an entry point to look differently depending on a define... So a developer of a multi-platform application would have to duplicate those ifdefs in his own code to use it.